public/lib/uploadProbeTest.js

```diff
@@ -32,6 +34,10 @@
             testRun: this._testIndex
         });
         this._test.start(this.size, this._testIndex, getRandomString(this.size));
+        var self = this;
+        this.interval = setInterval(function () {
```

Comment:
self is not needed.
this.interval = setInterval(function () {
 this._monitor();
 }.bind(this), 100);